.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test
```

Comment:
- [ ] Its suppose to be test.md not test, also kindly remove your test.md file manually from your github files because you already added it, it cannot be removed by ignoring it in .gitignore.